src/util.ts

```diff
@@ -186,25 +183,25 @@ export async function handleTransactionFetch(
   networkType: string,
   address: string,
   opt?: FetchAllOptions,
-): Promise<[{ [result: string]: [] }, { [result: string]: [] }]> {
+): Promise<any[]> {
```

Comment:
Why was this (less specific) type change required?

Reply:
Missed this. Thanks!